x/profile/client/cli/query.go

```diff
@@ -0,0 +1,74 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/desmos-labs/desmos/x/profile/internal/types"
+	"github.com/spf13/cobra"
+)
+
+// GetQueryCmd adds the query commands
+func GetQueryCmd(cdc *codec.Codec) *cobra.Command {
+	profileQueryCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Querying commands for the profile module",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+	profileQueryCmd.AddCommand(flags.GetCommands(
+		GetCmdQueryProfile(cdc),
+		GetCmdQueryProfiles(cdc),
+	)...)
+	return profileQueryCmd
+}
+
+// GetCmdQueryProfile queries a profile
+func GetCmdQueryProfile(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "profile [address]",
+		Short: "Retrieve the profile having the specified user address, if any.",
```

Comment:
Probably we can change this command to make so that is takes the first argument and performs two different types of search: 

1. Considering the argument as the address
2. Considering the argument as the moniker

So I could do 

```
desmoscli query profile desmos1....
desmoscli query profile riccardo
```

And both will be valid. 

The documentation should be updated as well

Reply:
So let me know if I understood well, editing this command to both accept address or moniker right?